examples/examples/Gyro/robot.py

```diff
@@ -0,0 +1,57 @@
+'''
+Created on Jan 10, 2016
+
+@author: winterst
+'''
+import wpilib
+from wpilib.joystick import Joystick
+class MyRobot(wpilib.SampleRobot):
+    
+    def robotInit(self):
+        self.gyroChannel = 0 #analog input
+        
+        self.joystickChannel = 0; #usb number in DriverStation
+    
+        #hannels for motors
+        self.leftMotorChannel = 1
+        self.rightMotorChannel = 0
+        self.leftRearMotorChannel = 3
+        self.rightRearMotorChannel = 2
+    
+        self.angleSetpoint = 0.0;
+        self.pGain = 1; #propotional turning constant
+    
+        #gyro calibration constant, may need to be adjusted; 
+        #gyro value of 360 is set to correspond to one full revolution
+        self.voltsPerDegreePerSecond = .0128
+              
+        self.myRobot = wpilib.RobotDrive(wpilib.CANTalon(self.leftMotorChannel), wpilib.CANTalon(self.leftRearMotorChannel), wpilib.CANTalon(self.rightMotorChannel), wpilib.CANTalon(self.rightRearMotorChannel))
+        self.gyro = wpilib.Gyro(self.gyroChannel)
```

Comment:
There is no Gyro object in 2016, it's called AnalogGyro. Did you run this with the 2016 version of robotpy?